bitcoin-core-dev-tech/2022-10-10-misc.es.md

```diff
@@ -0,0 +1,42 @@
+
+El verdadero 0conf debería ser "sólo envía la clave privada". Eso es incluso más rápido que 0conf. Me pregunto por qué ese no fue el punto de esquematización en torno al cual la gente se organizó.
+
+# Redes de tráfico anónimo
+
+Anonimizador de paquetes usando un HSM: la privacidad del tráfico está protegida por la raíz de confianza del HSM. Utilizar un HSM para enrutar el tráfico anónimo. Esto podría diseñarse de forma que sea más fácil detectar si alguien está manipulando una red, o crear un incentivo o presión para hacer raíces de confianza más fuertemente protegidas, como raíces de confianza descentralizadas acordadas por grupos.
+
+# Firma a mano
+
+Una idea es cifrar/descifrar a mano y exportar el proceso de firma a un ordenador utilizando un cifrado totalmente homomórfico. Podrías sustituir el algoritmo de firma por una versión totalmente homomórfica del algoritmo de firma. O si se trata principalmente de cálculos de celosía, entonces puede ser posible hacerlo a mano. Digamos que obtienes un blob encriptado completamente homomórfico y entonces sólo hay un algoritmo que puede ejecutarse en él para escupir `0` o `1` para la verificación.
+
+¿Tal vez si tienes un programa completamente encriptado homomórficamente, y luego el programa no puede ser modificado eficientemente por otras personas? ¿Necesitas una prueba de conocimiento-cero de verificación de firma? ¿O habría que ejecutarla a mano? Necesitas una prueba de ejecución correcta que es difícil de evitar.
+
+¿Podrías encontrar una curva que fuera más fácil de hacer a mano? Quizá una curva binaria sobre un campo binario. El campo base en vez de ser un primo grande sería un campo de Galois con `2^256` elementos o algo así. No sé si las operaciones son más fáciles de hacer allí. Supongo que a los humanos les resultaría más fácil operar en base `10`. El material existente que tiene Andrew funciona sobre el campo de `32` elementos. Pero entonces hay maquinaria extra con la que trabajar...
+
+Sugerí hace mucho tiempo, ni idea de lo factible que es, utilizar ayudas mecánicas. Hay ordenadores que pueden evaluar las transformadas de Fourier. Hay algoritmos de multiplicación que se basan en transformadas de Fourier. Como máquinas de cuentas o hardware mecánico. Algo así tal vez podría hacerse funcionar. Es cuestión de tiempo. Evaluar una multiplicación escalar de un punto de curva implica algo así como mil aproximaciones de Fourier... cada punto es `10`, así que hay `100` de ellos, así que hay alrededor de `1.000`.
```

Comment:
I think is unnecessary to use the `` syntax for the numbers in the above 2 paragraphs.